sematic/examples/mnist/pytorch/train_eval.py

```diff
@@ -127,6 +129,12 @@ def test(model: nn.Module, device: torch.device, test_loader: DataLoader):
         }
     )
 
+    # We don't need every point to make a useful plot, so we'll subsample
+    # every nth row (with `reduction_factor` as n), targeting
+    # a size of about 2000 points.
+    reduction_factor = int(len(df) / _N_POINTS_PR_CURVE) + 1
+    df = df.iloc[::reduction_factor, :]
```

Comment:
Does this need to be propagated to the other AUC graphs we generate in examples?

Reply:
we don't appear to have any others that would have this problem